app/jobs/order_cycle_webhook_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Creates a webhook payload for an order cycle
+class OrderCycleWebhookJob < ApplicationJob
+  def perform(order_cycle_id, event)
+    order_cycle = OrderCycle.find(order_cycle_id)
+    data = order_cycle.slice(:id, :name).merge(
+      {
+        orders_open_at: order_cycle.orders_open_at&.to_s,
+        orders_close_at: order_cycle.orders_close_at&.to_s,
+        enterprise_id: order_cycle.coordinator_id,
+        enterprise_name: order_cycle.coordinator.name,
+      }
+    )
+
+    endpoint = order_cycle.coordinator.webhook_endpoint
```

Comment:
Interesting. I could imagine that it would be very useful to loop through all distributors in that order cycle. Maybe clarify with product. One odd case is that the coordinator doesn't need to be selling within the OC. So it's like two different events or subscriptions. The enterprise may want to choose to receive events as coordinator, as shop or as producer. So maybe this is just the easiest for now. Clarify with product?

Reply:
I didn't think of that, good idea.
Do you mean to clarify in in the #product-circle channel?

To start with, I'll raise the question with Lynne on the issue.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/issues/9616#issuecomment-1271011028